### PR TITLE
Cross platform post-install

### PR DESCRIPTION
**Pull request by volovyks**
Bash approach does not work on Windows

### CODE REVIEW

**Comment by volovyks** [comment]
@ailisp do you know what is happening with our CI? It has not started...

**Comment by ailisp** [comment]
I also can't see reason why it doesn't. Can be github action scheduler's issue? Let me try close and reopen

**Comment by ailisp** [comment]
Started this time. Weird

**Review by ailisp — APPROVED** [review]
Good multi platform compatibility support!

File: src/cli/post-install.ts [hunk 1]
```diff
@@ -1,53 +1,68 @@
...
-async function download(url: string, verbose = false) {
-  await executeCommand(`curl -LOf ${url}`, verbose);
-}
+const PLATFORM = os.platform();
+const ARCH = os.arch();
+console.log(`Current platform: ${PLATFORM}, current architecture: ${ARCH}`);
 
-const OS = await executeCommand("uname -s", false);
-const ARCH = await executeCommand("uname -m", false);
+const SUPPORTED_PLATFORMS = ["linux", "darwin"]; // Unsaported platforms: 'win32', 'aix', 'freebsd', 'openbsd', 'sunos', 'android'
```
Comment (ailisp) [inline src/cli/post-install.ts:14]: Oh, changes below are made for windows support, but there's still some part of the build that doesn't support windows. Am I right?

Comment (volovyks) [reply src/cli/post-install.ts:14]: It's just part of the effort. So Windows is not on the list just yet.

**Pull request merged**